src/payment/cybersource.md

```diff
@@ -13,71 +13,89 @@ For details about making a secure transition from deprecated payment integration
 
 CyberSource supports shipments to [multiple addresses]({% link shipping/shipping-multiaddress.md %}) as part of the checkout flow. The order is duplicated for each address that the customer wants to ship to.
 
-## Step 1: Get Your CyberSource Credentials
+## CyberSource account credentials
 
-Sign up for a CyberSource [merchant account][2], and get your credentials.
+Sign up for a CyberSource [merchant account][2]. You will need the account credentials to connect your account in the Magento Admin.
 
-## Step 2: Enable CyberSource
+## Set up CyberSource in Magento
+
+{:.bs-callout-tip}
+Click <span class="btn">Save Config</span> at any time to save your progress.
+
+### Step 1: Enable the CyberSource payment method
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the panel on the left under **Sales**, choose **Payment Methods**. Under **Other Payment Methods**, expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **CyberSource (Deprecated)** section. Then, do the following:
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
 
-   - Set **Enabled** to `Yes`.
+1. If your Magento installation has multiple websites, stores, or views, set **Store View** to the store view where you want to apply this configuration.
 
-   - Accept the **Default Payment** action of `Authorized Only`, which approves the purchase and puts a hold on the funds. The amount is not withdrawn from the customer’s bank account until the sale is “captured” by the merchant.
+1. In the **Merchant Location** section, select the **Merchant Country** where your business is located.
 
-   - Enter a **Title** to identify CyberSource during checkout.
+   ![Merchant Country]({% link images/images/config-sales-payment-methods-merchant-location.png %}){: .zoom}
+   _Merchant Country_
+
+1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Deprecated Payment Methods** section and the **CyberSource (Deprecated)** section.
 
    ![Enable CyberSource]({% link images/images-ee/config-sales-payment-methods-cybersource1.png %}){: .zoom}
    _Enable CyberSource_
 
-## Step 3: Enter Your CyberSource Credentials
+1. Set **Enabled** to `Yes`.
+
+1. Accept the **Default Payment** action of `Authorized Only`, which approves the purchase and puts a hold on the funds. The amount is not withdrawn from the customer’s bank account until the sale is “captured” by the merchant.
```

Comment:
in a previous edit, `captured` was italic not in quotes. I'm not sure of the context here so maybe quotes are appropriate.

Reply:
let's be consistent and go with italic